tests/ui/imports/issue-114682-4.rs

```diff
@@ -1,12 +1,13 @@
-//@ check-pass
 //@ aux-build: issue-114682-4-extern.rs
 // https://github.com/rust-lang/rust/pull/114682#issuecomment-1880755441
 
 extern crate issue_114682_4_extern;
 
 use issue_114682_4_extern::*;
 
-fn a() -> Result<i32, ()> { // FIXME: `Result` should be identified as an ambiguous item.
+//~v ERROR type alias takes 1 generic argument but 2 generic arguments were supplied
```

Comment:
yeah see, this one picks one of the extern results and prints this error, that's how I'd expect things to work normally, the previous one with the traits in scope resolution was super inconsistent w/ my previous experience.